tasks/pcs-configure-pcs-pcsd.yml

```
@@ -19,6 +19,40 @@
   when:
     - ha_cluster_regenerate_keys
 
+- name: Create certificate and private_key files
```

```suggestion
- name: Create pcsd certificate and private_key files
```
Just to be clear about which certificate and key is being created. The role works with several pre-shared keys, and, once #68 merged, TLS certificates for qdevice. So it may get confusing.